web-ts/tools/tsconfig.webpack.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
Is this target correct even though we use `es2020` below?

Reply:
Should be es2020